### PR TITLE
Sqliteidx plugged

### PR DESCRIPTION
**Pull request by marian-vignau**
Some comments
- Deleted duplicated data of ptje. It was added ages ago to simplify debugging. It's not needed.
- This was the reason of most of changes in test_index.py. It was needed to avoid check ptje as last item in result.
- The API of easy_index, compressed_index and sqlite_index are not fully compatible. I renamed search to partial_search, and leave just a void function on search. 
- I changed the total max number of results provided by sqlite_index. In the GUI, there are a link that gives to the user 500 results by page, so that was the new limit.

Enjoy! #310

### CODE REVIEW

File: src/armado/sqlite_index.py [hunk 1]
```diff
@@ -376,11 +381,14 @@ def get_doc(self, docid):
         return row
 
     def search(self, keys):
+        pass
```
Comment (facundobatista) [inline src/armado/sqlite_index.py:384]: En vez del "pass" metele un docstring, algo como ....

```
"""Full search. 

No implementamos esta función porque la partial_search ya es lo suficientemente rápida entonces hicimos que el web server use directmaente esa.
"""
```

File: utilities/convert_index.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+# MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# For further info, check  https://github.com/PyAr/CDPedia/
+"""
+Creates a sqlite_index from a existing compressed_index
+"""
```
Comment (facundobatista) [inline utilities/convert_index.py:18]: Docstrings en imperativo, terminan con un ., las comillas en la misma linea.

`"""Do stuff."""`

File: utilities/search_index.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# -*- encoding: utf8 -*-
```
Comment (facundobatista) [inline utilities/search_index.py:1]: Esto no va...

**Review by facundobatista — APPROVED** [review]
Buenísimo! Ahí anoté un par de cambios menores....

File: utilities/convert_index.py (hunk 2 above)
Comment (marian-vignau) [reply utilities/convert_index.py:18]: Siempre meto la pata en el mismo ítem... jajajaaaaaaa vamos a necesitar un script que controle eso solo jajajaaaaaaa

**Pull request merged**